alethzero/plugins/accounts/AllAccounts.cpp

```diff
@@ -64,16 +66,75 @@ void AllAccounts::installWatches()
 {
 	aleth()->installWatch(ChainChangedFilter, [=](LocalisedLogEntries const&){ onAllChange(); });
 	aleth()->installWatch(PendingChangedFilter, [=](LocalisedLogEntries const&){ onAllChange(); });
+
+	connect(m_ui->accounts, SIGNAL(currentItemChanged(QListWidgetItem*,QListWidgetItem*)), SLOT(on_accounts_currentItemChanged()));
+
+	connect(m_ui->showBasic, &QCheckBox::toggled, [this]() { refresh(); });
+	connect(m_ui->showContracts, &QCheckBox::toggled, [this]() { refresh(); });
+	connect(m_ui->onlyKnown, &QCheckBox::toggled, [this]() { refresh(); });
+	connect(m_ui->accountsFilter, &QLineEdit::textChanged, [this]() { filterAndUpdateUi(); });
+	
+	auto copyText = [](QLabel* _label)
+	{
+		qApp->clipboard()->setText(_label->text());
+	};
+
+	connect(m_ui->addressCopy, &QPushButton::clicked, [&] () { copyText(m_ui->addressValue); });
+	connect(m_ui->ICAPCopy, &QPushButton::clicked, [&] () { copyText(m_ui->ICAPValue); });
+	connect(m_ui->codeCopy, &QPushButton::clicked, [&] () { copyText(m_ui->codeValue); });
+}
+
+void AllAccounts::create()
+{
+	m_ui = new Ui::AllAccounts;
```

Comment:
It would be cleaner to have a separate class `AllAccountsDialog : public QDialog` Please consider refactoring in the future